src/puppetlabs/dujour/version_check.clj

```diff
@@ -109,6 +111,14 @@
       (log/info update-msg))
     response))
 
+(defn- get-hash
+  "Returns a SHA-512 encoded value of the given string."
+  [data]
+  (let [md (. java.security.MessageDigest getInstance "sha-512")]
+    (. md update (.getBytes data))
+    (let [bytes (. md digest)]
```

Comment:
you're using two different syntaxes for java interop here, `.getBytes data` and `. md update ...` and it would probably be clearer with just one

Mostly peope in clojure stick to the one with the method name first and no space